src/pages/AuthPage/Auth.tsx

```diff
@@ -39,21 +42,23 @@ const Auth: React.FC<AuthProps> = (props: AuthProps) => {
 
   function handleGoodResponse(response: any) {
     if (authQuery === LOGIN_QUERY) {
-      dispatch(
-        setUser({
-          email: response.data.login.user.email,
-          id: response.data.login.user.id,
-          access_token: response.data.login.access_token
-        })
-      )
+      setUser({
+        email: response.data.login.user.email,
```

Comment:
you can do it more ellegant

Reply:
like so?
setUser({
        response.data.login.user.email,
        response.data.login.user.id,
        ...
      })

Reply:
like const {id, email} = response
setUser({
  id, email, ....
})